src/components/pages/products-list/ProductsGrid/ProductsGrid.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable max-len */
+import { productsData } from "../../../../mocks/products";
+import { categoriesData } from "../../../../mocks/product-categories";
+import { Styled } from "./Styled";
+
+import Card from "../../../common/Card";
+import Image from "../../../common/Image";
+import Title from "../../../common/Title";
+
+export function ProductsGrid({ activeFilters }) {
+  let activeList = [];
+  for (let filter in activeFilters) {
+    console.log(filter);
+    if (activeFilters[filter]) {
+      activeList.push(filter);
+    }
+  }
+
+  const categories =
+    (Object.keys(activeFilters).length === 0 &&
+    activeFilters.constructor === Object) || activeList.length === 0
+      ? categoriesData.results.map((category) => category.id)
+      : activeList;
```

Comment:
We can use the `activeList` variable for this condition, if the array has length it means the object `activeFilters` has attributes set to `true` and the categories in the array should be used, if not all categories should be displayed

```suggestion
  const categories = activeList.length === 0
    ? categoriesData.results.map(({ id }) => id)
    : activeList;
```